### DISCUSSION TITLE
TypeError when add caption to dp.File

### DISCUSSION THREAD

**Discussion by alisongh** [datapane/datapane#154] [Help] [answered]
I want to add a caption to an image like below.
`dp.File(file="./image/scatter_matrix.png", caption = "Figure 4")`

But I got the TypeError:
`TypeError: __init__() got an unexpected keyword argument 'caption'`

**Comment by johnmicahreid** [comment, marked as answer]
Hi Alison, thanks for bringing this up. We're working on a fix which will be deployed in the next couple of weeks!

**Reply by mands** [reply]
Fixed in 0.12.2 - please try it out and let us know how you get on!
